usagov/vote-gov
# [Vote-567] update link validation  tests

link to ticket -> https://bixal-projects.atlassian.net/browse/VOTE-567

This Pr updates the project's link validation tests in 2 ways.  It separates out internal and external links into 2 separate tests and then also adds the internal link check into the pipeline.  This PR also turns back on the link validation test in the pipeline.

Also an updated JSON for the site pages that adds in the homepage to be checked. 
to test:
- pull down branch 
- run `npm run start` 
- spilt terminal 
- run `npm run cy:open` from there you can select either of the link validation tests (note that the external will have some fails but that is expected after work and research done prior to this PR)
